docs/source/general/abstract_models.rst

```diff
@@ -35,25 +35,33 @@ Example:
 
    #django_freeradius/base/models.py
 
-   class AbstractRadiusGroup(TimeStampedEditableModel):
-       id = models.UUIDField(primary_key=True, db_column='id')
-       group_name = models.CharField(
-               verbose_name=_('groupname'), max_length=255, unique=True, db_column='groupname', db_index=True)
-       priority = models.IntegerField(verbose_name=_('priority'), default=1)
-       creation_date = models.DateField(verbose_name=_('creation date'), null=True, db_column='created_at')
-       modification_date = models.DateField(
-           verbose_name=_('modification date'), null=True, db_column='updated_at')
-       notes = models.CharField(
-           verbose_name=_('notes'), max_length=64, blank=True, null=True)
+    class BaseModel(TimeStampedEditableModel):
+    id = None
```

Comment:
indentation looks wrong here. what do you think?

Reply:
Yeah. Will fix. Thanks for pointing out.